src/common/options.cc

```diff
@@ -153,30 +153,83 @@ std::chrono::seconds parse_duration(const std::string& s)
   auto secs = 0s;
   size_t start = 0;
   size_t new_start = 0;
+  using years_t = duration<int, std::ratio<365 * 3600 * 24>>;
+  using months_t = duration<int, std::ratio<30 * 3600 * 24>>;
+  using weeks_t = duration<int, std::ratio<7 * 3600 * 24>>;
```

Comment:
could define `weeks_t` in term of `days_t`, like
```c++
using weeks_t = duration<int, std::ratio<7 * days_t::period::num>>;
```
better readability this way.